test/e2e/scenarios/remove_cluster.go

```diff
@@ -0,0 +1,13 @@
+package scenarios
+
+import (
+	. "github.com/onsi/ginkgo"
+)
+
+var _ = PDescribe("Removing policies related to removed cluster", func() {
+	PIt("Should remove policies relevant to removed cluster", func() {
+		By("Remove cluster2")
```

Comment:
I would leave cluster1 & 2 untouched to avoid affecting other tests.

I would

1) Add cluster 3, with pods/network policies that need to be processed
2) Verify that those network policies are created/processed
3) Remove cluster 3
4) Verify that those network policies are removed/processed/updated.